test/fixtures/is-builtin-module/case-3/after.js

```diff
@@ -0,0 +1,15 @@
+var {
+  isBuiltin
+} = require("node:module");
```

Comment:
Very annoyingly, jscodeshift/recast insert these new lines automatically. I've been trying to figure out how to prevent this without any luck.

I guess this PR is an improvement over the current state, which doesn't work, but it's still not ideal. I'd appreciate some help!

Reply:
I don't think its a huge problem, I think it's generally expected people run formatters after running codemods